ios/Podfile.lock

```diff
@@ -12,6 +12,37 @@ PODS:
     - React-Core (= 0.61.5)
     - React-jsi (= 0.61.5)
     - ReactCommon/turbomodule/core (= 0.61.5)
+  - Firebase/CoreOnly (6.25.0):
```

Comment:
@aledustet Is this ok to merge? Can't figure out how to get rid of all this stuff.